Packs/CarbonBlackEnterpriseEDR/Integrations/CarbonBlackEnterpriseEDR/CarbonBlackEnterpriseEDR.yml

```diff
@@ -1301,10 +1304,64 @@ script:
     - contextPath: CarbonBlackEEDR.SearchProcess.results.scriptload_count
       description: The cumulative count of loaded scripts since process tracking started.
       type: Number
-  dockerimage: demisto/python3:3.10.13.72123
+  - arguments:
+    - description: Threat ID to add the tags to.
+      isArray: true
+      name: threat_id
+      required: true
+    - description: The tags to be added to the threat.
+      name: tags
+      required: true
+    description: Update threat ID tags.
+    name: cb-eedr-add-threat-tags
+    outputs:
+    - contextPath: CarbonBlackEEDR.Threat.ThreatID
+      description: Threat ID.
+    - contextPath: CarbonBlackEEDR.Threat.Tags
+      description: Threat ID tags.
+  - arguments:
+    - description: Threat ID.
+      name: threat_id
+      required: true
+    - description: Notes to be added to the provided threat ID.
```

Comment:
Shouldn't this be a list as well?
If so let's add the "Comma-separated list of..." to the description and the isArray: true.

Reply:
It's a string.